### ISSUE TITLE
Sourcebans doesn't show CTBans

### ISSUE THREAD

**Issue by condolent** [GameConnect/SourceBans#81]
Sourcebans doesn't show CT bans.
I'm using this plugin to be able to CT ban people: https://forums.alliedmods.net/showthread.php?t=166080

**Comment by ErikMinekus** [comment]
CT Bans does not ban people, it just moves them from CT to T. Don't know why you expect them to show in SourceBans.

**Comment by condolent** [comment]
I've seen it on a server a while back.
Let me rephrase my original post: Is there a plugin or addon or something to SB so that can be shown?

**Comment by ErikMinekus** [comment]
You would have to ask in that topic if someone has that, we don't support third party plugins.
